tests/js/JSONSchemasInterface.tests.ts

```
@@ -23,7 +23,9 @@ describe("JSONSchemasInterface", () => {
         expect(schema?.properties?.inSet).to.be.an("object");
 
         if (
+            // @ts-ignore
```

Maybe just fail if schema is undefined instead or add it to if?

---

Right. Removed ifs